assets/styles/flairs.less

```diff
@@ -97,3 +97,23 @@ a[href="/lucky"], a[href="/egg"], a[href="/eevee"], a[href="/togepi"], a[href="/
 .flair2-beautyribbon:before { background-position: -44px 0;}
 .flair2-smartribbon:before { background-position: -66px 0; margin-bottom: -2px; }
 .flair2-toughribbon:before { background-position: -88px 0; margin-bottom: -2px; }
+
+.flair-rowlet:after, .flair-litten:after, .flair-popplio:after {
```

Comment:
If we're hardcoding the extra flairs into the stylesheet, I think we might as well hardcode them into the rest of the codebase as well, rather than putting an array in the config file.